buildroot-external/patches/occu/0094-WebUI-userSysVarTblValuesWordWrap.patch

```diff
@@ -0,0 +1,44 @@
+--- occu/WebUI/www/rega/esp/system.fn.orig
++++ occu/WebUI/www/rega/esp/system.fn
+@@ -1316,7 +1316,7 @@
+ 
+ ! [userId], [opt: sTmpVars]
+ function ::BuildUserSvTable() {
+-  WriteLine('<table cellpadding="3" cellspacing="1" class="CLASS02105" width="100%">');
++  WriteLine('<table cellpadding="3" cellspacing="1" class="CLASS02105 CLASS02105_RM" width="100%">');
+   WriteLine('<colgroup> <col style="width:15%;" /> <col style="width:20%;" /> <col style="width:15%;" /> <col style="width:20%;" /> <col style="width:15%;" /> <col style="width:15%;" />');
+   WriteLine('</colgroup> <tbody> <tr> <th class="CLASS02106">${thName}</th>');
+   WriteLine('<th class="CLASS02107">${thDescription}</th>');
+@@ -1358,7 +1358,7 @@
+         Write(s);
+         Write("</td>");
+         
+-        WriteLine("<td class='GrayBkg CLASS02110'>");
++        WriteLine("<td class='GrayBkg CLASS02110 CLASS02110_RM'>");
+         !if (sysvar.ValueSubType() == istPresent) {} !#### Auskommentierung z.Z. nicht erwünscht. Später KW.
+         
+         if (sysvar.ValueSubType() == istAlarm) {
+--- occu/WebUI/www/webui/style.css.orig
++++ occu/WebUI/www/webui/style.css
+@@ -4960,6 +4960,10 @@
+ 	background-color: $_(black); /* #000000 */
+ }
+ 
++.CLASS02105_RM {
```

Comment:
Please no extra `CLASSXXXX_RM` css classes but modify add your modifications to the class used there. AFAICS these class names are exclusive to these two uses.